packages/grafana-ui/src/components/Table/TableNG/components/MaybeWrapWithLink.tsx

```diff
@@ -23,8 +23,12 @@ export const MaybeWrapWithLink = memo(({ field, rowIdx, children }: MaybeWrapWit
   }
   // as faux link that acts as hit-area for tooltip activation
   else if (linksCount + actionsCount > 0) {
-    // eslint-disable-next-line jsx-a11y/anchor-is-valid
-    return <a aria-haspopup="menu">{children}</a>;
+    return (
+      // eslint-disable-next-line jsx-a11y/anchor-is-valid
+      <a title={t('table.link-wrapper.menu', 'view data links and actions')} aria-haspopup="menu">
```

Comment:
couldn't come up with a good way to target this guy with a testing-library test without adding a title, but it's also kind of nice to have a title on the link to help people understand what's gonna happen when they click.